### ISSUE TITLE
UglifyJs is not happy with versions after 2.3.8

### ISSUE THREAD

**Issue by Temeez** [apertureless/vue-chartjs#54]
Hey, there seems to be something odd in version 2.3.9 and onward when trying to run production mode (UglifyJs). After looking at the line the error below shows I see this: https://github.com/apertureless/vue-chartjs/blob/develop/src/BaseCharts/Line.js#L39 

### Expected Behavior
Not the **Actual Behavior**.

### Actual Behavior
```
`ERROR in app.js from UglifyJs
Unexpected token punc «(», expected punc «:» [app.js:36079,7]

ERROR in app.js from UglifyJs
Unexpected token punc «(», expected punc «:» [app.js:36079,7]
```

### Environment
- vue.js version: 2.2.1
- vue-chart.js version: >=2.3.9
- npm version: 3.10.8

**Comment by apertureless** [comment]
Hey, 
check the [release notes](https://github.com/apertureless/vue-chartjs/releases).

Files need to be transpiled first, because the main entry is set to the src files.
With Webpack 2 it should work out of the box.

With browserify you need babel presets.
Otherwise you can import the old dist files or create an alias for them.

**Comment by Temeez** [comment]
> Files need to be transpiled first, because the main entry is set to the src files.
I have no idea what this means, but I am using `"webpack": "^2.2.1"`. I also have the babel preset which my other stuff requires.

**Comment by apertureless** [comment]
It's fixed with `vue-chartjs@2.4.1` added additional transpiled es src. So webpack should get these instead of the untranspiled es7 sources.
